### DISCUSSION TITLE
customtkinter.set_appearance_mode("System") Is not working Correctly

### DISCUSSION THREAD

**Discussion by The-R4V3N** [TomSchimansky/CustomTkinter#1085] [General]
The appearance mode is not working correctly when i run the python file from Terminal or fram a make file. But it is working correctly from the run button in vs-code. My system is set to dark. So the Window should be dark and it should not matter how i start the script.

This shows upp when i run it inside vs-code:
![Calculator_dark](https://user-images.githubusercontent.com/18666373/212540983-ac0b4786-be8d-4c05-a249-48016009e650.png)

And this shows up when running from the terminal or when i start it via Makefile:
![Calculator_light](https://user-images.githubusercontent.com/18666373/212540989-1fef3912-46a3-4529-a04d-dbcb545017c0.png)

**Comment by ElectricCandlelight** [comment]
darkdetect~=0.3.1 is required for it to work, not sure how you have your makefile set up but you might want to make sure it's being added in

**Comment by The-R4V3N** [comment]
@ElectricCandlelight  Would you mid to explain how to include it. I am a beginner and learning all this with Makefiles and so on. Alternatively you could maybe give me a link where i can read how to include it. Thank you

**Comment by TomSchimansky** [comment]
darkdetect must be installed on the python version you are using in vscode.
